src/test/e2e/00_use_cli_test.go

```diff
@@ -82,5 +84,26 @@ func TestUseCLI(t *testing.T) {
 	require.NoError(t, err, "Error when reading image cache path")
 	assert.Greater(t, len(files), 1)
 
-	e2e.cleanFiles(shasumTestFilePath, imageCachePath, otherTmpPath)
+	pkgName = fmt.Sprintf("zarf-package-git-data-%s.tar.zst", e2e.arch)
+
+	// Pull once to test git cloning
+	stdOut, stdErr, err = e2e.execZarfCommand("package", "create", "examples/git-data", "--confirm", "--zarf-cache", cachePath, "--tmpdir", otherTmpPath, "--log-level=debug")
+	require.NoError(t, err, stdOut, stdErr)
+
+	files, err = os.ReadDir(gitCachePath)
+	require.NoError(t, err, "Error when reading git cache path")
+	assert.Greater(t, len(files), 1)
+
+	// Pull twice to test git fetching (from cache)
+	stdOut, stdErr, err = e2e.execZarfCommand("package", "create", "examples/git-data", "--confirm", "--zarf-cache", cachePath, "--tmpdir", otherTmpPath, "--log-level=debug")
+	require.NoError(t, err, stdOut, stdErr)
+
+	// Test removal of cache
+	stdOut, stdErr, err = e2e.execZarfCommand("tools", "clear-cache", "--zarf-cache", cachePath, "--log-level=debug")
+	require.NoError(t, err, stdOut, stdErr)
+
+	_, err = os.ReadDir(cachePath)
+	require.Error(t, err, cachePath+": no such file or directory")
```

Comment:
This display message seems to be what the expected err would be. Should the display message be more along the lines of `Did not receive expected error when reading a directory that should not exist`?

Reply:
I mixed up the args here.  Changed it to ErrorContains and added that text to be more explicit (and correct)